aspnetcore/security/authentication/configure-jwt-bearer-authentication.md

```diff
@@ -0,0 +1,327 @@
+---
+title: Configure JWT bearer authentication in ASP.NET Core
+author: damienbod
+description: Learn how to set up JWT bearer authentication in an ASP.NET Core app.
+monikerRange: '>= aspnetcore-8.0'
+ms.author: riande
+ms.custom: mvc
+ms.date: 12/7/2024
+uid: security/authentication/configure-jwt-bearer-authentication
+---
+# Configure JWT bearer authentication in ASP.NET Core
+
+By [Damien Bowden](https://github.com/damienbod)
+
+JWT (JSON Web Token) Bearer Authentication is commonly utilized for APIs. While it operates similarly to cookie authentication, instead of a cookie, the server issues a JWT or tokens upon a successful authentication. A JWT is a self-contained token that encapsulates information for an API resource or a client. The client which requested the JWT can request data from an API resource using the Authorization header and a bearer token.
```

Comment:
```suggestion
JWT (JSON Web Token) Bearer Authentication is commonly utilized for APIs. While it operates similarly to cookie authentication, instead of a cookie, the identity provider issues a JWT or tokens upon a successful authentication. A JWT is a self-contained token that encapsulates information for an API resource or a client. The client which requested the JWT can request data from an API resource using the Authorization header and a bearer token.
```

I don't want to imply that the server running ASP.NET Core has built-in support to issue JWTs.